### DISCUSSION TITLE
[FEATURE REQUEST] Para las construcciones de rutina, mostrar cuál es la que falta un campo obligatorio por completar

### DISCUSSION THREAD

**Discussion by NicolasPLG** [Cotalker/documentation#141] [3. Ideas & Feature Requests] [answered]
Hola todos

Les comento lo siguiente: Estaba creando una rutina con unos network request y unos condicionales. Arreglé un campo en la condicional que  me arrojaba error y al ver, no puedo guardar.

Reviso la rutina y veo todos los campos de forma correcta. La pedida es la siguiente:

1.- Sería bueno mostrar o marcar qué rutina es la que está generando error, ya que visualmente no sabes qué campo de la rutina es el que está teniendo problemas. Actualmente sólo queda actualizar el navegador y construir la rutina paso a paso

Adjunto una imagen de lo que digo + propuesta

![image](https://user-images.githubusercontent.com/61419471/116940406-ed58cd80-ac3b-11eb-9717-91cbee17394e.png)

Gracias

**Comment by acedward** [comment, marked as answer]
¿Pudiste encontrar el campo que faltaba finalmente?

Si suena importante poder encontrar donde está el problema, creo que es un buen feature request.

**Reply by NicolasPLG** [reply]
hola @acedward al final no lo pude encontrar :(, asi que tendré que hacer de nuevo la rutina.

**Reply by acedward** [reply]
No hay más feedback. Voy a cerrar.
